daemon/inertiad/build/builder.go

```diff
@@ -299,7 +312,7 @@ func (b *Builder) herokuishBuild(d Config, cli *docker.Client,
 	}
 
 	// Start the herokuish container to build project
-	fmt.Fprintln(out, "Building project...")
+	fmt.Fprintf(out, "Building project %s...\n", d.Name)
```

Comment:
Perhaps you'd want to put duplicate strings in a var for consistency

Reply:
I was considering that, but wasn't sure if it would be within the scope of the PR - though considering theres a bunch of misc changes, might as well